csi/exporter.py

```diff
@@ -44,6 +45,21 @@ def collect(self):
             'Kadalu Storage Inodes free Count',
             labels=capacity_labels
         )
+        subvol_capacity_bytes = GaugeMetricFamily(
+            'kadalu_storage_subvol_capacity_bytes',
```

Comment:
Instead of `_subvol_` change it to `_pv_` so that it will be familiar name. `subvol` is internal to Gluster/Kadalu